things-to-org.py

```diff
@@ -6,16 +6,16 @@
 from datetime import datetime
 
 def get_todo_keyword(task, suffix=' '):
-    if task['type'] == 'project':
-        return ''
+    # if task['type'] == 'project':
+    #     return ''
```

Comment:
Deleted, because it should also create a "TODO" for projects (a project is a TODO with sub-tasks in my Org).